commands/bmc.nicole

```diff
@@ -182,24 +180,30 @@ function cmd_datetime_show {
   format_date "${compact}" "${utc}" "$(date +%s)"
 }
 
+function call_netconfig() {
+  local FNAME=${FUNCNAME[1]##cmd_}
+  local PNAME="bmc ${FNAME//_/ }"
+  exec -a "${PNAME}" netconfig --cli "$@"
+}
+
 # @sudo cmd_datetime_ntpconfig admin
 # @restrict cmd_datetime_ntpconfig admin
 # @doc cmd_datetime_ntpconfig
 # Configure NTP server connection
-#  {add|del} IP
+#  {add|del} ADDR [ADDR..]
 function cmd_datetime_ntpconfig {
-  if [[ $# -ne 2 ]]; then
-    abort_badarg "Expected exactly 2 arguments"
+  if [[ $# -lt 2 ]]; then
+    abort_badarg "Expected 2 or more arguments"
   fi
-  netconfig ntp "eth0" "$1" "$2"
+  call_netconfig ntp "eth0" "$1" "$2"
```

Comment:
I think here is an error.
I guess `call_netconfig ntp eth0 $@` should be here if we want pass several addresses.

Reply:
Good catch! (Although that should have been fixed by you when you added this functionality to netconfig ;) )

Reply:
I have a task about this, but I haven't started to solve it yet.